BezierKit/MacDemos/Demos.swift

```diff
@@ -466,6 +466,44 @@ class Demos {
                                 }
 
     })
+    static let demo23 = Demo(title: "PathComponent(arcCenter: ...)",
+                             quadraticControlPoints: [],
+                             cubicControlPoints: [],
+                             drawFunction: {(context: CGContext, demoState: DemoState) in
+                                if demoState.quadratic {
+                                    return
+                                }
+                                
+                                let pathComponent = PathComponent(
+                                    arcCenter: CGPoint(x: 100, y: 105),
+                                    radius: 100,
+                                    startAngle: .pi / 2,
+                                    endAngle: 2 * .pi,
+                                    clockwise: true
+                                )
+                                Draw.drawPathComponent(context, pathComponent: pathComponent)
+                                for curve in pathComponent.curves {
+                                    Draw.drawSkeleton(context, curve: curve)
+                                }
+    })
+    static let demo24 = Demo(title: "PathComponent(ovalIn rect: CGRect)",
+                             quadraticControlPoints: [],
+                             cubicControlPoints: [],
+                             drawFunction: {(context: CGContext, demoState: DemoState) in
+                                if demoState.quadratic {
+                                    return
+                                }
+                                
+                                let pathComponent = PathComponent(
+                                    ovalIn: CGRect(
+                                        x: 0, y: 52.5,
+                                        width: 200, height: 100
+                                    )
+                                )
+                                Draw.drawPathComponent(context, pathComponent: pathComponent)
+                                Draw.setColor(context, color: Draw.pinkish)
+                                Draw.drawBoundingBox(context, boundingBox: pathComponent.boundingBox)
+    })
```

Comment:
let's try to make this demo more interactive. We could let the user drag the origin and scale the oval perhaps